app/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM node:20 as dependencies
+WORKDIR usr/src/app
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)